commands/download_kubeseal.go

```diff
@@ -0,0 +1,68 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	kubesealLatestReleaseURL = "https://api.github.com/repos/bitnami-labs/sealed-secrets/releases/latest"
+	msg                      = `# Add $HOME/.openfaas/bin/ to your $PATH variable
+
+export PATH=$PATH:$HOME/.openfaas/bin`
+)
+
+func init() {
+	downloadCmd.AddCommand(downloadKubeSeal)
+}
+
+var downloadKubeSeal = &cobra.Command{
+	Use:     `kubeseal`,
+	Short:   "Download kubeseal",
+	Long:    "Download kubeseal",
```

Comment:
I think we can expand this a little bit too
```suggestion
	Long:    "Download the kubeseal cli. This is needed to create and use sealed secrets in OpenFaaS Cloud",
```

Maybe add a link to the docs, https://docs.openfaas.com/openfaas-cloud/secrets/? what do you think @alexellis  and @rgee0

Reply:
Well it could be changed to `faas-cli cloud seal --download` instead? Would that self-document?